src/components/custom-field-input-number/custom-field-input-number.test.jsx

```diff
@@ -58,12 +58,12 @@ describe('CustomFieldInputNumber', () => {
       const { rerender } = render(<CustomFieldInputNumber {...requiredProps} errorText="" />);
       expect(screen.queryByText("Here's some help text!")).not.toBeInTheDocument();
       rerender(<CustomFieldInputNumber {...requiredProps} errorText="Here's some help text!" />);
-      expect(screen.queryByText("Here's some help text!")).toBeInTheDocument();
+      expect(screen.getByTestId('error-message')).toBeInTheDocument();
     });
 
     it('shows the provided errorText when true', () => {
       render(<CustomFieldInputNumber {...requiredProps} errorText="Here's some help text!" />);
-      expect(screen.getByText("Here's some help text!")).toBeInTheDocument();
+      expect(screen.getByTestId('error-message')).toBeInTheDocument();
```

Comment:
Follow the same patterns as the other tests you wrote in this PR: use `expect(...).toHaveDescription(...)`